CONTRIBUTING.md

```diff
@@ -4,10 +4,10 @@ You can add a record using any of the following three methods. We have a few met
 
 - Complete PR
   - add record in [README](https://github.com/opensourcecities/montreal/blob/master/README.md) AND
-  - add record's `yml` file (based on [Type of Records](#type-of-records)) in `data` folder
```

Comment:
Using `yml` in `data` folder was the old way, and now it's changed to `md` file in `content` folder, can you adjust file format in your PR as well (`yml` -> `md`)?